Add Sega Pico

n2qz opened the pull request.

dmanlfc commented:
have you tested all cores & they work with no configgen modifications?

also add: changelog
you don't need the update to `systems-explanations.yml`

dmanlfc commented:
`name: Sega Pico`

n2qz commented:
> have you tested all cores & they work with no configgen modifications?

They all work within their limitations, e.g. there's no control of the stylus with libretro-picodrive.

> also add: changelog 

Added.

> you don't need the update to `systems-explanations.yml`

Removed.

n2qz commented:
> `name: Sega Pico`

Fixed.

dmanlfc commented:
> > have you tested all cores & they work with no configgen modifications?
> 
> They all work within their limitations, e.g. there's no control of the stylus with libretro-picodrive.
> 
> > also add: changelog
> 
> Added.
> 
> > you don't need the update to `systems-explanations.yml`
> 
> Removed.

The other two have stylus control???

n2qz commented:
> The other two have stylus control???

Yes.  In the genesisplusgx* emulators the mouse moves the stylus, left mouse button is stylus button, right mouse button is the red button, and mouse scroll wheel turns the pages.

With libretro-picodrive, the pages can be turned with the controller shoulder buttons.

dmanlfc commented:
@n2qz sorry can you resolve the conflict so i can merge, thanks

zognic commented:
Sega Pico system need to be lock to LR PicoDrive core only.
